README.md

```diff
@@ -6,6 +6,8 @@
 
 This action enables you to run code analysis with Parasoft dotTEST and review analysis results directly on GitHub.
 
+Parasoft dotTEST is a testing tool that automates software quality practices for .NET applications. It uses a comprehensive set of analysis techniques, including pattern-based static analysis, dataflow analysis, metrics, code coverage, and unit testing to help you verify code quality and ensure compliance with industry standards, such as CWE or OWASP.
+
 Parasoft dotTEST is a testing tool that automates software quality practices for C# and VB.NET applications. 
```

Comment:
Nasty repetition: Parasoft dotTEST is a testing tool that automates software quality practices for